### PR TITLE
style: buils-card

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- ( ) Refactor
- (X) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

### CODE REVIEW

File: apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts [hunk 1]
```diff
@@ -29,6 +30,15 @@ import { CreateCommunityBuildComponent } from './components/create-community-bui
 import { BuildListItemComponent } from './components/my-community-builds/build-list-item/build-list-item.component';
 import { MyCommunityBuildsComponent } from './components/my-community-builds/my-community-builds.component';
 import { TeammateInviteConfirmationComponent } from './components/teammate-invite-confirmation/teammate-invite-confirmation.component';
+import { PublicHomeListEventsModule } from 'apps/commudle-admin/src/app/feature-modules/listing-pages/public-home-list-events/public-home-list-events.module';
+import { FeaturedProjectsComponent } from './components/community-builds/featured-projects/featured-projects.component';
+import { FeaturedProjectsCardComponent } from '../../app-shared-components/featured-projects-card/featured-projects-card.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts]: let's give relative path while import

File: apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts [hunk 2]
```diff
@@ -29,6 +30,15 @@ import { CreateCommunityBuildComponent } from './components/create-community-bui
...
+import { BuildsHeaderComponent } from 'apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/builds-header/builds-header.component';
+import { BuildsTopBuildersComponent } from './components/community-builds/builds-top-builders/builds-top-builders.component';
+import { TopBuildersCardComponent } from '../../app-shared-components/top-builders-card/top-builders-card.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts]: let's give relative path while import

File: apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts [hunk 3]
```diff
@@ -29,6 +30,15 @@ import { CreateCommunityBuildComponent } from './components/create-community-bui
...
+import { ExploreExpertsComponent } from './components/community-builds/explore-experts/explore-experts.component';
+import { BuildsComponent } from './components/community-builds/builds/builds.component';
+import { SkeletonCardsComponent } from '../skeleton-screens/components/skeleton-cards/skeleton-cards.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts]: let's give relative path while import

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.scss [hunk 4]
```diff
@@ -3,51 +3,43 @@ p {
...
+      @apply com-flex;
+      .community-build-card-image {
+        @apply com-flex com-justify-center com-items-center md:com-w-[135px] md:com-h-[90px] com-w-[102px] com-h-[68px] com-overflow-hidden;
+
+        // .upcoming-card-image {
+        //   @apply com-border-r-gray-200 com-flex com-justify-center com-items-center com-overflow-hidden md:com-w-[388px] md:com-h-[145px] com-w-full com-h-full;
+        // }
+        // img {
+        //   @apply com-w-full com-h-full com-object-contain com-rounded;
+        // }
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.scss]: let's remove this if not needed!

File: apps/shared-components/tag/tag.component.ts [hunk 5]
```diff
@@ -29,7 +31,7 @@ export class TagComponent implements OnInit, OnDestroy {
   }
 
   getTags() {
-    return this.tags.filter(Boolean);
+    return this.tags.filter(Boolean).slice(0, 5);
```
Comment (ArshdeepGrover) [inline apps/shared-components/tag/tag.component.ts]: we are fixing this to 5 why?

File: apps/shared-components/tag/tag.component.ts [hunk 6]
```diff
@@ -12,6 +12,8 @@ export class TagComponent implements OnInit, OnDestroy {
   @Input() editable: boolean;
   @Input() inputDisabled: boolean;
   @Input() minimumTags: number = 5;
+  @Input() customStyle: boolean = false;
```
Comment (ArshdeepGrover) [inline apps/shared-components/tag/tag.component.ts]: Why are we using this?

File: apps/shared-models/user.model.ts [hunk 7]
```diff
@@ -39,6 +39,9 @@ export interface IUser {
   user_roles_users?: UserRolesUsers;
   deactivated: boolean;
   speaker_events_count: number;
+  total_builds_votes: number;
+  published_community_builds_count: number;
+  communities_count: number;
```
Comment (ArshdeepGrover) [inline apps/shared-models/user.model.ts]: let's make this optional

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/app-shared-components/top-builders-card/top-builders-card.component.scss [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+          .username {
+            @apply md:com-text-sm com-text-xs com-font-normal com-line-clamp-1 com-text-Quartz com-mt-0 com-mx-0 com-mb-1;
+          }
+
+          .votes {
+            @apply com-text-sm com-font-normal com-m-0 com-text-Quartz;
+          }
+
+          .designation {
+            @apply md:com-text-sm com-text-xs com-font-normal com-mx-0 com-mb-1 com-text-Dark-Jungle-Green com-line-clamp-2;
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/top-builders-card/top-builders-card.component.scss:40]: Is dark jungle green actually green or grey?

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/builds-header/builds-header.component.ts [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+@Component({
+  selector: 'commudle-builds-header',
+  templateUrl: './builds-header.component.html',
+  styleUrls: ['./builds-header.component.scss'],
+})
+export class BuildsHeaderComponent implements OnInit {
+  buildsPageHeader: IListingPageHeader;
+  richText: string;
+
+  constructor(private cmsService: CmsService) {}
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/builds-header/builds-header.component.ts:14]: Should we rename this service to sanity cms service?

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/builds-top-builders/builds-top-builders.component.html [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+      <div class="text">
+        <h3 class="title">TOP BUILDERS</h3>
+        <h4 class="sub-heading">Top Builders you can check out</h4>
+      </div>
+    </div>
+    <div class="tags">
+      <nb-tag-list>
+        <nb-tag
+          appearance="outline"
+          [text]="options[0]"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/builds-top-builders/builds-top-builders.component.html:16]: options is not a very good variable name

**Pull request merged**